python/ray/exceptions.py

```diff
@@ -257,6 +257,33 @@ class RayActorError(RayError):
     This exception could happen either because the actor process dies while
     executing a task, or because a task is submitted to a dead actor.
 
+    If the actor is dead because of an exception thrown in its creation tasks,
+    RayActorError will contain the creation_task_error, which is used to
+    reconstruct the exception on the caller side.
```

Comment:
What attribute is the `creation_task_error` stored in?

Reply:
Looks like that attribute is removed in https://github.com/ray-project/ray/commit/5514711a3506f9e2675786b13b78c8f10a008f34 . I can revisit this in another PR.